npm/qsharp/src/shared/circuit.ts

```diff
@@ -52,15 +71,17 @@ export interface Operation {
   /** Formatted gate arguments to be displayed. */
   displayArgs?: string;
   /** Nested operations within this operation. */
-  children?: Operation[];
+  children?: Operation[][];
+  /** Number of columns to span. */
```

Comment:
Consider saying in the comment what the default is if this property is omitted. 1?